completion_test.py

```diff
@@ -8,23 +8,27 @@
 
 
 messages = [{ "content": "Hello, how are you?","role": "user"}]
-# openai call
-response = completion(model="gpt-3.5-turbo", messages=messages)
-print("\nOpenAI call")
-print(response)
-
-# azure openai call
-response = completion("chatgpt-test", messages, azure=True)
-print("\nAzure call")
-print(response)
-
-# text davinci openai call
-response = completion("text-davinci-003", messages)
-print("\nDavinci call")
-print(response)
-
-# cohere call
-response = completion("command-nightly", messages)
-print("\nCohere call")
-print(response)
-
+# # openai call
+# response = completion(model="gpt-3.5-turbo", messages=messages)
```

Comment:
can we leave all the other tests uncommented

Reply:
Will do in ~15 minutes. Thanks for reviewing 🧡

Reply:
Uncommented + deleted the unneeded testos.py (my bad) 🥲

One question - I could try implementing everything using pattern matching. Should I try that, or we'll lose backward compatibility this way?

Reply:
let's address pattern matching in a separate PR